tests/e2e/state.go

```diff
@@ -26,8 +26,7 @@ type ChainState struct {
 	ConsumerChains                 *map[chainID]bool
 	AssignedKeys                   *map[validatorID]string
 	ProviderKeys                   *map[validatorID]string // validatorID: validator provider key
-	ConsumerChainQueueSizes        *map[chainID]uint
-	GlobalSlashQueueSize           *uint
+	PendingPacketQueueSize         *uint                   // Only relevant to consumer chains
```

Comment:
Can we make the name more explicitly mention Consumer chains again?
```suggestion
	ConsumerChainsPendingPacketQueueSize         *uint
```

Reply:
https://github.com/cosmos/interchain-security/commit/3ea0cf4626129c9a5db0800c44cc6d56389dd181